etc/docs/README.md

```diff
@@ -0,0 +1,38 @@
+# docs_utils
+
+This directory contains scripts to generate api docs as well our the Hugo site template used for the MongoDB node driver documentation.
+
+There are two scripts contained in this folder.
+
+- `legacy-generate.sh` was used to generate API documentation before the driver's docs
+were moved into the main repository.  This script has the ability to generate api docs for older versions of the driver (in case it becomes
+necessary to backport a feature).
+
+- `generate-docs.ts` is used to generate API docs for a major or minor release.
+
+### Dependencies
+
+`generate-docs.ts` requires the following in addition to dependencies installed with `npm i`:
+
+* Hugo static web generator `v0.30.2`
+  * You can download the right version [here](https://github.com/gohugoio/hugo/releases/tag/v0.30.2)
+* ts-node
+
+Note: `typedoc` is also a dependency but it is downloaded by the docs generation script automatically.
+
+`legacy-generate.sh` requires the following (in addition to Hugo):
+
+* jsdoc v3
+* node (v6.x or v8.x) and npm (>= 3.x). Note: worked for me with 8.17.0, but not 8.0.0.
+* python sphinx
+
+### Usage
+
+To generate API documentation for a new major or minor version:
+
+`npm run docs:build -- <version> <optional status for the version>`
+
+The generated docs can be previewed using `npm run docs:preview`.
+
+Once everything looks correct, open a PR against `main`.  Once the PR is merged, the docs will automatically re-build via a Github action.  Once this
```

Comment:
Just a question - is there an actual github action that *we* configure to rebuild the site or does it just automatically get updated once we point the github pages setting in the repo to the main docs branch? If there is no action for us to manually manage, it may be better just to say here that our github pages config points to the /docs directory on main and is kept up to date by github itself

Reply:
The Github action is managed by Github and it should update as soon as we switch the location of the static site hosting from `gh-pages` to `main/docs`.  I'll update the wording here